include/Imap.php

```diff
@@ -74,11 +74,8 @@ class Imap implements ImapInterface
      */
     public function open($mailbox, $username, $password, $options = 0, $n_retries = 0, array $params = null)
     {
-        $state = new StateSaver();
-        $state->pushErrorLevel();
-        error_reporting(0);
```

Comment:
🤔 Should we keep this error reporting stuff?

Reply:
iirc by default the ```pushErrorLevel()``` function actually immediately logs not to use it and to remove ```error_reporting``` from your code. Imo we should remove it if we can.